packages/components/src/Label/Label.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { LabelHTMLAttributes } from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./Label.module.scss"
+type LabelType = "text" | "checkbox" | "toggle" | "radio"
+
+export type LabelProps = OverrideClassName<
+  LabelHTMLAttributes<HTMLLabelElement>
+> & {
```

Comment:
Just a suggestion, but if you put `OverrideClassName<LabelHTMLAttributes<HTMLLabelElement>>` after the `{}`, I think it doesn't break into multiline.

Reply:
Hm, looks fine to me 😄